Makefile

```diff
@@ -82,17 +82,20 @@ ifeq ("$(BUILD_COMPAT)", "1")
 TESTS += ccurl-multi_pow
 endif
 
-TESTS += remote-new-task
-TESTS += remote-get-result
-
 TESTS := $(addprefix $(OUT)/test-, $(TESTS))
 
+MANUNAL_TESTS = \
```

Comment:
Don't call it "manual tests", which sounds a bit sophisticated. Instead, we can use "trial", which implies the fashion of manual operations.